### DISCUSSION TITLE
Dual Technic Hub Connection for passing data and control

### DISCUSSION THREAD

**Discussion by wolbar** [pybricks/support#280] [Ideas]
Is there a possibility that two technic hubs can be connected to the browser simultanously, and therefor pass data back and forth to each other via the browser? This would be a very good way to expand the capability of these hubs as two hubs can be mounted in the same robot. I know that data can be sent back to the browser, but can new data be read into a variable which changed the operating program? And can a program in a second hub remain suspended until a status change initiated by a change in a browser variable?

**Comment by laurensvalk** [comment]
Check out #262 for some of our plans. Do you think this would cover your use case?

**Comment by wolbar** [comment]
Thanks it has been awhile!  I tried the beta firmware and could not get two Technic Hubs to communicate. I posted an example of the error message but for some reason it was not saved. l will get back on this in the new year. Thank you for your continued attention to multi-hub comms and Happy New Year!

**Reply by massimiliano-mantione** [reply]
@wolbar Could you please check [this](https://github.com/pybricks/support/discussions/890) discussion, particularly follow [this](https://github.com/pybricks/pybricks-micropython/pull/80#issuecomment-1367108766) link, and check if you are experiencing the same problem I am seeing?
